diff --git a/boost.sh b/boost.sh
index ebb98be..e3bf063 100755
--- a/boost.sh
+++ b/boost.sh
@@ -531,6 +531,11 @@ parseArgs()
                 CXX_FLAGS="$CXX_FLAGS -fvisibility=hidden -fvisibility-inlines-hidden"
                 ;;
 
+            --cxxflags)
+                CXX_FLAGS="$CXX_FLAGS $2"
+                shift
+                ;;
+
             --universal)
                 UNIVERSAL=1
                 ;;
@@ -690,11 +695,7 @@ version() { echo "$@" | awk -F. '{ printf("%d%03d%03d%03d\n", $1,$2,$3,$4); }';
 
 downloadBoost()
 {
-    if [ "$(version "$BOOST_VERSION")" -ge "$(version "1.63.0")" ]; then
-        DOWNLOAD_SRC=https://boostorg.jfrog.io/artifactory/main/release/${BOOST_VERSION}/source/boost_${BOOST_VERSION_SNAKE_CASE}.tar.bz2
-    else
-        DOWNLOAD_SRC=http://sourceforge.net/projects/boost/files/boost/${BOOST_VERSION}/boost_${BOOST_VERSION_SNAKE_CASE}.tar.bz2/download
-    fi
+    DOWNLOAD_SRC=https://archives.boost.io/release/${BOOST_VERSION}/source/boost_${BOOST_VERSION_SNAKE_CASE}.tar.bz2
     if [ ! -s "$BOOST_TARBALL" ]; then
         echo "Downloading boost ${BOOST_VERSION} from ${DOWNLOAD_SRC}"
         curl -L -o "$BOOST_TARBALL" "$DOWNLOAD_SRC"
