--- tools/build/src/engine/build.orig.sh	2025-09-09 13:26:17
+++ tools/build/src/engine/build.sh	2025-09-09 13:34:12
@@ -391,8 +391,8 @@

     clang*)
         CXX_VERSION_OPT=${CXX_VERSION_OPT:---version}
-        B2_CXXFLAGS_RELEASE="-O3 -s"
-        B2_CXXFLAGS_DEBUG="-O0 -fno-inline -g"
+        B2_CXXFLAGS_RELEASE="-O3 -s -Wno-deprecated-declarations"
+        B2_CXXFLAGS_DEBUG="-O0 -fno-inline -g -Wno-deprecated-declarations"
     ;;

     tru64cxx)
